experiments/compositions/composition/expanders/jinja2/main.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+
+	pb "google.com/composition/proto"
+	"google.golang.org/grpc"
+	"tailscale.com/atomicfile"
+)
+
+var (
+	port = flag.Int("port", 50051, "The server port")
+)
+
+// server is used to implement exander.Evaluator
+type server struct {
+	pb.UnimplementedExpanderServer
+}
+
+type expander struct {
+	path string
+	req  *pb.EvaluateRequest
+}
+
+// Verify the expander config/template
+func (s *server) Validate(context.Context, *pb.ValidateRequest) (*pb.ValidateResult, error) {
+	return &pb.ValidateResult{Status: pb.Status_SUCCESS}, nil
+}
+
+// Evaluate the expander config in context of inputs and return manifests
+func (s *server) Evaluate(ctx context.Context, req *pb.EvaluateRequest) (*pb.EvaluateResult, error) {
+	result := &pb.EvaluateResult{
+		Status:    pb.Status_SUCCESS,
+		Errors:    []string{},
+		Type:      pb.ResultType_MANIFESTS,
+		Manifests: []byte{},
+		Values:    []byte{},
+	}
+
+	log.Printf("EvaluateRequest:\n  config: %s\n  context: %s\n  facade: %s\n  values: %s\n", req.Config, req.Context, req.Facade, req.Value)
+	dir, err := os.MkdirTemp("", "jinja2")
+	if err != nil {
+		newerr := fmt.Errorf("Unexpected tmp file creation failure. %v", err)
```

Comment:
Two nits:

Probably should use : %w
go errors should not start with a capital

Reply:
changes all errors to %w.
removed capitalization for error strings.